treeherder/model/management/commands/import_perf_data.py

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.core.management.base import BaseCommand, CommandError
+from optparse import make_option
+from treeherder.client import PerfherderClient
+from treeherder.model.derived.jobs import JobsModel
+
+import concurrent.futures
+import json
+
+
+def _add_series(project, time_interval, signature_hash, signature_props,
+                mysql_debug):
+    with JobsModel(project) as jm:
+        jm.DEBUG = mysql_debug
+        signature_property_placeholders = []
+        for (k, v) in signature_props.iteritems():
+            if k == 'subtest_signatures':
+                v = json.dumps(v)
+            signature_property_placeholders.append([
+                str(signature_hash), str(k), str(v),
+                str(signature_hash), str(k), str(v),
+            ])
+
+        jm.jobs_execute(
+            proc='jobs.inserts.set_series_signature',
+            placeholders=signature_property_placeholders,
+            executemany=True)
+
+        pc = PerfherderClient()
```

Comment:
I'm a bit confused here... why are you storing the series signature directly to the db while the performance series is posted with the Perfherder Client?